Modules/QR Reader/MITScannerMgr.m

```diff
@@ -86,7 +88,9 @@ - (void)setupCaptureSession
     self.deviceInput = [[AVCaptureDeviceInput alloc] initWithDevice:self.captureDevice error:&error];
     if( self.deviceInput == nil )
```

Comment:
@yevgmotov This is not a valid way to check the privacy settings of the camera. The device will not be nil if the user has refused to grant the app access to the camera. You should be using ```+[AVCaptureDevice authorizationStatusForMediaType:]``` instead.